integrations/huggingface-transformers/README.md

```diff
@@ -34,87 +30,156 @@ Coming soon!
+The following command will use the 80% sparse-quantized BERT model from the SparseZoo and fine-tune it on the SQuAD dataset, resulting in a model that achieves an F1 of 0.885 on the validation set. Keep in mind that the `--distill_teacher` argument is set to pull a dense SQuAD model from the SparseZoo to enable it to run independent of the dense teacher step. If you trained a dense teacher, change this out for the path to your model folder:
 
-For example, the following command can be run from within the integration's folder to export a trained/sparsified model's checkpoint:
 ```bash
-python transformers/examples/pytorch/question-answering/run_qa.py \
-  --model_name_or_path MODELS_DIR/bert-base-12layers_prune80 \
-  --dataset_name squad \
-  --do_eval \
-  --per_device_eval_batch_size 64 \
-  --max_seq_length 384 \
-  --doc_stride 128 \
-  --output_dir MODELS_DIR/bert-base-12layers_prune80/eval \
-  --cache_dir cache \
-  --preprocessing_num_workers 6 \
-  --onnx_export_path MODELS_DIR/bert-base-12layers_prune80/onnx
+sparseml.transformers.question_answering \
+    --output_dir models/sparse_quantized \
+    --model_name_or_path zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/12layer_pruned80_quant-none-vnni \
+    --recipe zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/12layer_pruned80_quant-none-vnni?recipe_type=transfer-question_answering \
+    --distill_teacher zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none \
+    --dataset_name squad \
+    --per_device_train_batch_size 12 \
+    --per_device_eval_batch_size 24 \
+    --preprocessing_num_workers 6 \
+    --do_train \
+    --do_eval \
+    --evaluation_strategy epoch \
+    --fp16 \
+    --seed 21636  \
+    --per_device_train_batch_size 16 \
+    --per_device_eval_batch_size 24 \
+    --preprocessing_num_workers 6 \
+    --save_strategy epoch \
+    --save_total_limit 1
 ```
 
-The DeepSparse Engine [accepts ONNX formats](https://docs.neuralmagic.com/sparseml/source/onnx_export.html) and is engineered to significantly speed up inference on CPUs for the sparsified models from this integration.
-Examples for loading, benchmarking, and deploying can be found in the [DeepSparse repository here](https://github.com/neuralmagic/deepsparse).
+### Exporting to ONNX
+
+The DeepSparse Engine uses the ONNX format to load neural networks and then deliver breakthrough performance for CPUs by leveraging the sparsity and quantization within a network.
+
+You will first export the trained BERT model to an ONNX format. The SparseML installation additionally provided a `sparseml.transformers.export_onnx` command. You will use this to load the training model folder and create a new model.onnx file within. Be sure the `--model_path` argument points to your trained model. By default, it is set to the result from transfer learning a sparse-quantized BERT model: 
+
+    --model_path "models/sparse_quantized".
```

Comment:
this seem extraneous with the command right after

Reply:
simplified the text